vmdb/spec/tools/fix_auth/fix_auth_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "spec_helper"
+
+$LOAD_PATH << Rails.root.join("tools")
+
+require "fix_auth/fix_auth"
+require "fix_auth/models"
+
+describe FixAuth::FixAuth do
+  describe "#fix_database_yml" do
+    it "respects --hardcode" do
+      subject = described_class.new(:password => 'newpass', :root => '/', :databaseyml => true)
+      expect(FixAuth::FixDatabaseYml).to receive(:run).with(:hardcode => 'newpass')
+      subject.fix_database_yml
+    end
+
+    it "respects --password" do
```

Comment:
this one should clearer that it's an alias or something `support --password password as an alias of --hardcode for ease of use`